Software/scan_key/on_off.py

```diff
@@ -52,3 +52,10 @@ def blinkKey():
     ledOn(blue)
     t.sleep(.1)
     ledOff(blue)
+
+def blinkKey2():
```

Comment:
Maybe we should consider an alternate framework for LED flashing.

Perhaps one function with a  few different strobe patterns and a parameter to specify the color.